cmd/common/image/build.go

```diff
@@ -49,7 +49,7 @@ import (
 var makefile []byte
 
 const (
-	DEFAULT_BUILDER_IMAGE = "ghcr.io/inspektor-gadget/inspektor-gadget-ebpf-builder:latest"
+	DEFAULT_BUILDER_IMAGE = "ghcr.io/inspektor-gadget/ebpf-builder:latest"
```

Comment:
Once https://github.com/inspektor-gadget/inspektor-gadget/issues/1420 is fixed, `DEFAULT_BUILDER_IMAGE` could be set to a tag such as "ghcr.io/inspektor-gadget/ebpf-builder:v0.21.0" if cmd/common/version.go defines the variable `version` as a release.

In that way, the old versions of `ig` would not break when a new version update the ebpf-builder image with some incompatible command line.

This problem existed before this PR, so I'll approve it.